src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  line-height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline-start: 0;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style-type: none;
```

Comment:
Styling using tag names like 'list-style-type: none;' on '.nav__item' can potentially lead to conflicts if you decide to use list items outside of navigation. It's better to move this style to the '.nav__list' class.